feedthefox/base/templates/jinja2/profile.html

```diff
@@ -85,38 +85,72 @@
     </div>
   </div>
 
-</div>
-
-<!-- Device Modal -->
-<div class="modal fade" id="DeviceModal" tabindex="-1" role="dialog" aria-labelledby="DeviceModal">
-  <div class="modal-dialog" role="document">
-    <div class="modal-content">
-      <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title" id="DeviceModalLabel">Add device</h4>
-      </div>
-      <form action="" method="POST">
-        {% csrf_token %}
-        <div class="modal-body">
-          <label>Device</label>
-          <div class="form-group">
-            {{ field_with_attrs(device_info_form.device, class='form-control') }}
-            {{ device_info_form.device.errors }}
-          </div>
-          <div class="form-group">
-            <label>IMEI</label>
-            {{ field_with_attrs(device_info_form.imei, class='form-control') }}
-            {{ device_info_form.imei.errors }}
+  <!-- Device Modal -->
+  <div class="modal fade" id="DeviceModal" tabindex="-1" role="dialog" aria-labelledby="DeviceModal">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="DeviceModalLabel">Add device</h4>
+        </div>
+        <form action="" method="POST">
+          {% csrf_token %}
+          <div class="modal-body">
+            <label>Device</label>
+            <div class="form-group">
+              {{ field_with_attrs(device_info_form.device, class='form-control') }}
+              {{ device_info_form.device.errors }}
+            </div>
+            <div class="form-group">
+              <label>IMEI</label>
+              {{ field_with_attrs(device_info_form.imei, class='form-control') }}
+              {{ device_info_form.imei.errors }}
+            </div>
+            <div class="form-group imei-tos">
+              <p>
+                By providing my device IMEI, I certify and acknowledge to Mozilla (i.e., Mozilla
+                Foundation or Mozilla Corporation, including any applicable local affiliate) that:
+              </p>
+              <ul>
+                <li>
```

Comment:
It might make sense to use larger fonts for ``By providing ...`` and then smaller for the rules.